### ISSUE TITLE
Nextron 6.0.1 webpack configuration broken

### ISSUE THREAD

**Issue by dac09** [saltyshiomix/nextron#136]
Hi, I just tried updating to the latest nextron. Relevant dependencies

```
    "next": "^10.0.3",
    "nextron": "^6.0.1",
```
It seems to fail launching with a Babel error. 
```
App threw an error during load
Error: Module parse failed: Unexpected token (51:12)
```

My project is based on [with-typescript-emotion](https://github.com/saltyshiomix/nextron/blob/master/examples/with-typescript-emotion)

**Comment by saltyshiomix** [comment]
Hi @dac09 !

> My project is based on with-typescript-emotion

So you mean, did you upgrade nextron from v5 to v6?

**Comment by dac09** [comment]
Yes, that's right.

**Comment by saltyshiomix** [comment]
Sorry I didn't write upgrade guides.

**Comment by saltyshiomix** [comment]
Do you have `.babelrc` ?

**Comment by saltyshiomix** [comment]
As of `nextron@6`, we need to set presets to use custom `.babelrc` for main process:

```json
{
  "presets": [
    "nextron/babel"
  ]
}
```

**Comment by dac09** [comment]
Unfortunately that still doesn't work

Updated `.babelrc` with the following
```
{
  "presets": ["nextron/babel", "next/babel", "@emotion/babel-preset-css-prop"],
  "plugins": ["emotion", "babel-plugin-macros"]
}
```

**Comment by dac09** [comment]
Sorry, I updated the wrong babel rc in `renderer/.babelrc`. However adding .babelrc in the root of the project doesn't seem to work either.

**Comment by saltyshiomix** [comment]
No problem, thank you for your reporting the issue!

I'll investigate it, please wait a moment.

**Comment by saltyshiomix** [comment]
@dac09 

Do you have `./.babelrc`?

I didn't encounter the problem...

**Comment by dac09** [comment]
Yes I tried with it too, same issue

**Comment by saltyshiomix** [comment]
@dac09 

Could you tell me the code `L51:12`?

```
Error: Module parse failed: Unexpected token (51:12)
```

**Comment by dac09** [comment]
Nothing special @saltyshiomix, just some `import` statements. I don't have the exactly log now, but it definitely works on nextron 5.x

**Comment by jarthursantos** [comment]
I'm having the same problem, i just create a `.babelrc` with `"presets": ["nextron/babel", "next/babel"]` and this don't work, in `5.x` a log pops up in console informing external babel config, but now that don't occours

**Comment by dac09** [comment]
Closing this, as it seems to have been resolved in `6.0.6`. I was able to upgrade and build without too much hassle!
